### PR TITLE
fix: Add MongoDB 'name cannot be null' troubleshooting steps

### PR DESCRIPTION
**Pull request by spark-c**
Added steps on the [query troubleshoorting page](https://docs.appsmith.com/help-and-support/troubleshooting-guide/query-errors) to troubleshoot MongoDB error that gives the error message `Name cannot be null`.

I wasn't able to reproduce this specific error message within Appsmith (I got other error messages that told me to add the database name), however here are some user support queries that reference this:
- [Includes error message and screenshot](https://app.intercom.com/a/inbox/y10e7138/inbox/shared/all/conversation/164629100153137)
- [User query solved by this advice](https://app.intercom.com/a/inbox/y10e7138/inbox/shared/all/conversation/164629100150172#part_id=initial-part-164629100150172-1382878289)
- [No conclusive solution, but mentions the error](https://app.intercom.com/a/inbox/y10e7138/inbox/shared/all/conversation/164629100076361)

Please let me know if I'm understanding the cause of the error correctly, since I wasn't able to reproduce it!

Resolves [appsmithorg/appsmith#16038]

### CODE REVIEW

**Review by jnikhila — DISMISSED** [review]

**Review by danciaclara — COMMENTED** [review]
Just check with one of the engineers from the Integrations pod for this if we are not entirely sure of the error message @rohan-arthur can you help us with this? 

@jnikhila Let's wait before we merge this, so @spark-c can use the the same Error message component that we are working on here.

**Comment by spark-c** [comment]
Hey @danciaclara just checking in about the Error message component you mentioned above. I'm not 100% about what that is, let me know if any action is needed on my part! :)

**Comment by jnikhila** [comment]
@spark-c - I've added a new component that would highlight the error message. See [PR#561](https://github.com/appsmithorg/appsmith-docs/pull/561). Once that PR is merged, you'll have to modify the error message to be added as a `Message` tag.

File: website/docs/help-and-support/troubleshooting-guide/query-errors.md [hunk 1]
```diff
@@ -9,15 +9,19 @@ The section illustrates common query errors and how to resolve them on Appsmith.
...
-You could see the query/API execution fails and generates an error response. 
+You could see the query/API execution fails and generates an error response:
 
-> **[<QUERY_OR_API_NAME>] action returned an error response. Response size exceeded the maximum supported size of <SIZE_SPECIFIED_IN_FILE> MB. Please use LIMIT to reduce the amount of data fetched.**
+<Message
+ messageContainerClassName="error"
+messageContent="<QUERY_OR_API_NAME> action returned an error response. Response size exceeded the maximum supported size of <SIZE_SPECIFIED_IN_FILE> MB. Please use LIMIT to reduce the amount of data fetched."></Message>
 
 
 ![Response larger than the supported size](/img/Query-errors-response-size-larger-than-5MB.png)
```
Comment (jnikhila) [inline website/docs/help-and-support/troubleshooting-guide/query-errors.md:19]: Let's add the screenshot in Cause section

File: website/docs/help-and-support/troubleshooting-guide/query-errors.md [hunk 2]
```diff
@@ -9,15 +9,19 @@ The section illustrates common query errors and how to resolve them on Appsmith.
...
 
 #### Error message
-Response size exceeded the maximum supported size of <SIZE_SPECIFIED_IN_FILE> MB. Please use LIMIT to reduce the amount of data fetched.
+<Message
+ messageContainerClassName='error'
+messageContent='Response size exceeded the maximum supported size of <SIZE_SPECIFIED_IN_FILE> MB. Please use LIMIT to reduce the amount of data fetched.'></Message>
 
 ![Response larger than the supported size error shown in errors tab](/img/Query-errors-response-size-larger-than-5MB-errors-tab.png)
```
Comment (jnikhila) [inline website/docs/help-and-support/troubleshooting-guide/query-errors.md:26]: Let's add the screenshot in Cause section

**Review by jnikhila — CHANGES_REQUESTED** [review]

File: website/docs/help-and-support/troubleshooting-guide/query-errors.md (hunk 2 above)
Comment (spark-c) [reply website/docs/help-and-support/troubleshooting-guide/query-errors.md]: @jnikhila moved both!

**Review by jnikhila — APPROVED** [review]

**Pull request merged**